### PR TITLE
Fix for DTMapGenerator and cleanup for DTTTrigCorrectionFirst (in CalibMuon/DTCalibration)

### PR DESCRIPTION
**Pull request by perrotta**
#### PR description:

Dead assignment reports of the static analyzer pointed out at something which looks like a bug (with a rather obvious fix) in the endJob() methid of CalibMuon/DTCalibration/plugins/DTMapGenerator.cc.
Perhaps @battibass can confirm

Another dead assignment in CalibMuon/DTCalibration/plugins/DTTTrigCorrectionFirst.cc can be fixed simply by removing the useless code.

#### PR validation:

Code builds

### CODE REVIEW

**Comment by perrotta** [comment]
please test

File: CalibMuon/DTCalibration/plugins/DTMapGenerator.cc [hunk 1]
```diff
@@ -130,8 +128,7 @@ void DTMapGenerator::endJob() {
             } else {
               outRob = robCounter + 21;
             }
-          }
-          if (sector == 4) {
+          } else if (sector == 4) {
```
Comment (perrotta) [inline CalibMuon/DTCalibration/plugins/DTMapGenerator.cc:131]: This is the actual bugfix: otherwise `outRob` assignments above get overwritten

**Comment by tvami** [comment]
+alca
 * bugfix
 * test diffs are coming from different (known) sources

**Comment by tvami** [comment]
type bug-fix

**Comment by perrotta** [comment]
+1

**Pull request merged**